app/src/main/java/ch/epfl/sdp/appart/database/DatabaseService.java

```diff
@@ -115,6 +115,9 @@
     @NonNull
     CompletableFuture<Boolean> putImage(Uri uri, String name, String path);
```

Comment:
I think this function should have the signature : 
`CompletableFuture<Boolean> putImage(Uri uri, String path);` with the name of the image included. That's how it feels the more natural in my opinion. I know you did not wrote this but we should talk about this in the next meeting.

Reply:
I feel the same way, tomorrow we can discuss about this and choose, so that next sprint we can change these little things

Reply:
Personally I agree (I decided deleteImage signature this way). It's a small refactor, I think everyone is going to see it more natural, I will include it in this PR.